src/components/tabs/demoDynamicTabs/index.html

```diff
@@ -16,13 +16,25 @@
 
   <form ng-submit="addTab(tTitle,tContent)" flex>
     <div layout="row" layout-sm="column" layout-margin layout-align="left center">
-      <md-text-float label="Active Index" ng-model="selectedIndex" disabled></md-text-float>
-      <md-text-float label="Active Title" ng-model="tabs[selectedIndex].title"></md-text-float>
+      <md-input-container>
+        <label>Active Index</label>
+        <input type="text" ng-model="selectedIndex">
```

Comment:
I definitely think you should make it disabled. The fact that it does not work in the current docs, is a bug (and one of the main reasons for implementing `md-input-container`; i.e. the fact that `md-text-float` did not propagate the attributes to the `input`).